src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -98,28 +99,9 @@ public void cleanRepoDirectory() throws IOException {
         FileUtil.deleteDirectory(FileUtil.REPOS_ADDRESS);
     }
 
-    @Test
-    public void validateFormats_alphaNumeric_success()
-            throws InvalidLocationException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
-        Method m = RepoConfiguration.class.getDeclaredMethod("validateFormats", List.class);
-        m.setAccessible(true);
-        m.invoke(null, Arrays.asList("java", "7z"));
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void validateFormats_nonAlphaNumeric_throwIllegalArgumentException() throws Throwable {
-        try {
-            Method m = RepoConfiguration.class.getDeclaredMethod("validateFormats", List.class);
-            m.setAccessible(true);
-            m.invoke(null, Arrays.asList(".java"));
-        } catch (InvocationTargetException ite) {
-            throw ite.getCause();
-        }
-    }
-
     @Test
     public void repoConfig_usesStandaloneConfig_success() throws InvalidLocationException, GitDownloaderException {
-        RepoConfiguration actualConfig = new RepoConfiguration(TEST_REPO_DELTA, "master");
+        RepoConfiguration actualConfig = new RepoConfiguration(new Location(TEST_REPO_DELTA), "master");
```

Comment:
Why not make it a Location?

`i.e.` `private static final Location TEST_REPO_DELTA = new Location(...)`

or if the string is required for validation

`private static final String TEST_REPO_DELTA = "..."`
`private static final Location TEST_REPO_DELTA_LOCATION = new Location(TEST_REPO_DELTA)`